pkgs/development/python-modules/earcut/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  numpy,
+  fetchPypi,
+  setuptools,
+  buildPythonPackage,
+}:
+
+buildPythonPackage rec {
+  pname = "earcut";
+  version = "1.1.5";
+  pyproject = true;
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-IcVbAy/guQZY5/MF4e5AjLT3Z4LFBm2GJzrftLbMI+A=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    numpy
+  ];
+
+  pythonImportsCheck = [ "earcut" ];
+
+  meta = {
+    description = "A pure Python port of the earcut JS triangulation library";
```

```suggestion
    description = "Pure Python port of the earcut JS triangulation library";
```
> Not start with definite (“The”) or indefinite (“A”/“An”) article.

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description